<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>NetworkAudit</class>
 <widget class="QMainWindow" name="NetworkAudit">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>672</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTabWidget" name="tab1">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>791</width>
      <height>571</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="Devices">
     <attribute name="title">
      <string>Devices</string>
     </attribute>
     <widget class="QGroupBox" name="System">
      <property name="geometry">
       <rect>
        <x>150</x>
        <y>10</y>
        <width>241</width>
        <height>171</height>
       </rect>
      </property>
      <property name="title">
       <string>System Information</string>
      </property>
      <widget class="QPlainTextEdit" name="system_info">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>20</y>
         <width>221</width>
         <height>141</height>
        </rect>
       </property>
       <property name="readOnly">
        <bool>true</bool>
       </property>
      </widget>
     </widget>
     <widget class="QGroupBox" name="Processor">
      <property name="geometry">
       <rect>
        <x>400</x>
        <y>10</y>
        <width>261</width>
        <height>171</height>
       </rect>
      </property>
      <property name="title">
       <string>Processor Information</string>
      </property>
      <widget class="QPlainTextEdit" name="processor_info">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>20</y>
         <width>241</width>
         <height>141</height>
        </rect>
       </property>
       <property name="readOnly">
        <bool>true</bool>
       </property>
      </widget>
     </widget>
     <widget class="QGroupBox" name="Graphics">
      <property name="geometry">
       <rect>
        <x>150</x>
        <y>190</y>
        <width>241</width>
        <height>171</height>
       </rect>
      </property>
      <property name="title">
       <string>Graphics Information</string>
      </property>
      <widget class="QPlainTextEdit" name="graphics_info">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>20</y>
         <width>221</width>
         <height>141</height>
        </rect>
       </property>
       <property name="readOnly">
        <bool>true</bool>
       </property>
      </widget>
     </widget>
     <widget class="QGroupBox" name="Sound">
      <property name="geometry">
       <rect>
        <x>400</x>
        <y>190</y>
        <width>261</width>
        <height>171</height>
       </rect>
      </property>
      <property name="title">
       <string>Sound Information</string>
      </property>
      <widget class="QPlainTextEdit" name="sound_info">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>20</y>
         <width>241</width>
         <height>141</height>
        </rect>
       </property>
       <property name="readOnly">
        <bool>true</bool>
       </property>
      </widget>
     </widget>
     <widget class="QGroupBox" name="Memory">
      <property name="geometry">
       <rect>
        <x>150</x>
        <y>370</y>
        <width>241</width>
        <height>161</height>
       </rect>
      </property>
      <property name="title">
       <string>Memory Information</string>
      </property>
      <widget class="QPlainTextEdit" name="memory_info">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>20</y>
         <width>221</width>
         <height>131</height>
        </rect>
       </property>
       <property name="readOnly">
        <bool>true</bool>
       </property>
      </widget>
     </widget>
     <widget class="QGroupBox" name="Network">
      <property name="geometry">
       <rect>
        <x>400</x>
        <y>370</y>
        <width>261</width>
        <height>161</height>
       </rect>
      </property>
      <property name="title">
       <string>Network information</string>
      </property>
      <widget class="QPlainTextEdit" name="network_info">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>20</y>
         <width>241</width>
         <height>131</height>
        </rect>
       </property>
       <property name="readOnly">
        <bool>true</bool>
       </property>
      </widget>
     </widget>
     <widget class="QPushButton" name="scan_button">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>490</y>
        <width>131</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Scan Network</string>
      </property>
     </widget>
     <widget class="QListWidget" name="device_list">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>131</width>
        <height>471</height>
       </rect>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="Charts">
     <attribute name="title">
      <string>Charts</string>
     </attribute>
     <widget class="GraphicsLayoutWidget" name="chart_grid" native="true">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>661</width>
        <height>531</height>
       </rect>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="Credentials">
     <attribute name="title">
      <string>Credentials</string>
     </attribute>
     <widget class="QGroupBox" name="groupBox">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>20</y>
        <width>251</width>
        <height>191</height>
       </rect>
      </property>
      <property name="title">
       <string>Credentals</string>
      </property>
      <widget class="QPlainTextEdit" name="plainTextEdit">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>20</y>
         <width>231</width>
         <height>31</height>
        </rect>
       </property>
      </widget>
      <widget class="QPlainTextEdit" name="plainTextEdit_2">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>80</y>
         <width>231</width>
         <height>31</height>
        </rect>
       </property>
      </widget>
     </widget>
     <widget class="QGroupBox" name="groupBox_2">
      <property name="geometry">
       <rect>
        <x>310</x>
        <y>20</y>
        <width>251</width>
        <height>191</height>
       </rect>
      </property>
      <property name="title">
       <string>SSH</string>
      </property>
      <widget class="QPlainTextEdit" name="plainTextEdit_3">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>20</y>
         <width>231</width>
         <height>31</height>
        </rect>
       </property>
      </widget>
      <widget class="QPlainTextEdit" name="plainTextEdit_4">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>80</y>
         <width>231</width>
         <height>31</height>
        </rect>
       </property>
      </widget>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>672</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>GraphicsLayoutWidget</class>
   <extends>QWidget</extends>
   <header>pyqtgraph</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
